pages/common/dot.md

```diff
@@ -0,0 +1,11 @@
+# dot
+
+> A command-line tool to produce layered drawings of directed graphs.
+
+- Use output filenames based on input filename and format:
+
+`dot -Tpng -O {{path/to/file.dot}}`
+
+- Create an SVG from DOT file:
+
+`dot -Tsvg -o{{path/to/out_file.svg}} {{path/to/file.dot}}`
```

Comment:
There aren't any long versions of these flags? We prefer them if possible :smiley_cat:

Reply:
@sbrl sadly, there are no long versions :(

Reply:
No problem :-)